docs/Prism.html

```diff
@@ -1210,13 +1210,7 @@ <h4 class="name" id=".tokenize"><span class="type-signature">(static) </span>tok
 
     <h5>Example</h5>
     
-    <pre class="prettyprint"><code>let code = `var foo = 0;`;
-let tokens = Prism.tokenize(code, Prism.languages.javascript);
-tokens.forEach(token => {
-    if (token instanceof Prism.Token &amp;&amp; token.type === 'number') {
-        console.log(`Found numeric literal: ${token.content}`);
-    }
-});</code></pre>
+    <pre class="prettyprint"><code>let code = `var foo = 0;`;
let tokens = Prism.tokenize(code, Prism.languages.javascript);
tokens.forEach(token => {
    if (token instanceof Prism.Token &amp;&amp; token.type === 'number') {
        console.log(`Found numeric literal: ${token.content}`);
    }
});</code></pre>
```

Comment:
I have [seen this before](https://github.com/PrismJS/prism/pull/2505#discussion_r468180842). I hoped this was an one-of-a-kind occurrence but it seems to be systematic.

May I ask what OS you're using?

Reply:
> May I ask what OS you're using?
Windows 10

Reply:
> Windows 10

Same. So it's not that.

I don't want this to hold back your pull any longer but one last question: What line ends are used in this section of the file specifically? It should be all line feeds and look like this:

![image](https://user-images.githubusercontent.com/20878432/90388339-ed9e4200-e087-11ea-93a7-cd59f07f95c2.png)

Reply:
CR LF
![image](https://user-images.githubusercontent.com/19926084/90388829-1d057c80-e091-11ea-993b-759b1eed646c.png)

Reply:
For Docs/prism.html
![image](https://user-images.githubusercontent.com/19926084/90389143-b2087580-e091-11ea-81fd-3aa787b713f3.png)

Reply:
Thank you for the information!

So it actually generates the `CR`s... I'll look into that some more but only after we merge this. Please revert the changes to the `docs/**` files and rebuild to make the CI pass.